app/src/common/constants/localization/eventsLocalization.js

```diff
@@ -128,11 +128,11 @@ export const actionMessages = defineMessages({
   },
   [ASSIGN_USER]: {
     id: 'EventActions.assignUser',
-    defaultMessage: 'Assign',
+    defaultMessage: 'Assign user',
```

Comment:
Could you please also change this message in localization files?

The same for Unassign user

Reply:
Done.